vllm/model_executor/layers/attention/backends/xformers.py

```diff
@@ -182,41 +145,117 @@ def forward(
             )
 
         # Reshape the output tensor.
-        return output.view(batch_size, seq_len, hidden_size)
+        return output.view(-1, self.num_heads * self.head_size)
+
+    def _multi_query_kv_attention(
```

Comment:
I think people can confuse the method name with multi-query attention (MQA). IIRC, this is the old name we used previously. I named it and deleted the method after I found people confused.